R/edit.R

```diff
@@ -43,8 +43,11 @@ edit_r_makevars <- function(scope = c("user", "project")) {
 edit_git_config <- function(scope = c("user", "project")) {
   scope <- match.arg(scope)
   switch(scope,
-    user = edit_file("~", ".gitconfig"),
-    project = edit_file(proj_get(), ".git/config")
+    user = edit_file(scope_dir(scope, git = TRUE), ".gitconfig"),
+    project = {
+      create_directory(proj_get(), ".git")
```

Comment:
It's either this or condition on `uses_git()`.

Reply:
FWIW, if you pre-create valid entries in `.git/config` THEN do `git init`, they are preserved (and augmented). So I guess it's OK to not require a git repo.